systest/backup/advanced-scenarios/127-Namespace/backup_test.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/dgraph-io/dgraph/systest/backup/common"
+	"github.com/dgraph-io/dgraph/testutil"
+)
+
+var (
+	restoreLocation = "/data/backups/"
+	backupDst       = "/data/backups/"
+	localBackupDest = "./data/backups"
+	headerAlpha1    = http.Header{}
+	headerAlpha2    = http.Header{}
+)
+
+const (
+	accessJwtHeader = "X-Dgraph-AccessToken"
+)
+
+func TestDeletedNamespaceID(t *testing.T) {
+	err := common.RemoveContentsOfPerticularDir(t, "./data/backups/")
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+
+	jwtTokenAlpha1 := testutil.GrootHttpLogin("http://" + testutil.SockAddrHttp + "/admin").AccessJwt
+	headerAlpha1.Set(accessJwtHeader, jwtTokenAlpha1)
+
+	jwtTokenAlpha2 := testutil.GrootHttpLogin("http://" + testutil.ContainerAddr("alpha2", 8080) + "/admin").AccessJwt
+	headerAlpha2.Set(accessJwtHeader, jwtTokenAlpha2)
+
+	common.AddNamespaces(t, 20, headerAlpha1, "")
```

Comment:
we should not ignore the returned values

Reply:
AddNamespaces() functions return namespaceId which will be used in deleted namespace test. (we pass 4the namespaceId to DeleteNamespace() function)

127-namespace test doesn't consume this id anywhere in the code, so I am ignoring the returned value.

Reply:
I think you could always use underscore so that we know we are explicitly ignoring the returned values